src/storage.ts

```diff
@@ -46,6 +46,9 @@ export interface CreateBucketQuery {
 }
 
 export interface StorageOptions extends ServiceOptions {
+  backoffMultiplier?: number;
```

Comment:
Have we considered factoring the retry options into some sort of sub-config? Not sure if this is a thing in Node. In Go for example the various mathy configs are all in https://pkg.go.dev/github.com/googleapis/gax-go/v2#Backoff